From 49014f240ab0dec63445f4a88d9c3df92c2eca7d Mon Sep 17 00:00:00 2001
Message-ID: <49014f240ab0dec63445f4a88d9c3df92c2eca7d.1696804840.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1696804840.git.sharlatanus@gmail.com>
References: <cover.1696804840.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 7 Oct 2023 17:51:34 +0100
Subject: [PATCH 14/20] gnu: python-stcal: Update package style.

* gnu/packages/astronomy.scm (python-stcal): Update indentation to
improve readability of the package.
---
 gnu/packages/astronomy.scm | 53 +++++++++++++++++++++-----------------
 1 file changed, 29 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 2a32b4e182..ee15394ed7 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2545,32 +2545,37 @@ (define-public python-stcal
   (package
     (name "python-stcal")
     (version "1.4.4")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "stcal" version))
-              (sha256
-               (base32
-                "031ldihdmsdrwz4wl49bfk2bxmzbp1i3kidrw46xz936765zmnc0"))))
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "stcal" version))
+       (sha256
+        (base32 "031ldihdmsdrwz4wl49bfk2bxmzbp1i3kidrw46xz936765zmnc0"))))
     (build-system pyproject-build-system)
     (arguments
-     (list #:phases #~(modify-phases %standard-phases
-                        ;; XXX: Can't detect opencv-python version. The input
-                        ;; opencv might not set the version correctly.
-                        (delete 'sanity-check)
-                        (add-before 'check 'build-extensions
-                          (lambda _
-                            ;; Cython extensions have to be built before
-                            ;; running the tests.
-                            (invoke "python" "setup.py" "build_ext" "--inplace"))))))
-    (propagated-inputs (list opencv ;Provides OpenCV-Python
-                             python-astropy python-numpy python-scipy))
-    (native-inputs (list python-cython
-                         python-psutil
-                         python-pytest
-                         python-pytest-cov
-                         python-pytest-doctestplus
-                         python-pytest-openfiles
-                         python-setuptools-scm))
+     (list
+      #:phases
+      #~(modify-phases %standard-phases
+          ;; XXX: Can't detect opencv-python version. The input opencv might
+          ;; not set the version correctly.
+          (delete 'sanity-check)
+          (add-before 'check 'build-extensions
+            (lambda _
+              ;; Cython extensions have to be built before running the tests.
+              (invoke "python" "setup.py" "build_ext" "--inplace"))))))
+    (propagated-inputs
+     (list opencv ;Provides OpenCV-Python
+           python-astropy
+           python-numpy
+           python-scipy))
+    (native-inputs
+     (list python-cython
+           python-psutil
+           python-pytest
+           python-pytest-cov
+           python-pytest-doctestplus
+           python-pytest-openfiles
+           python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/stcal")
     (synopsis "STScI tools and algorithms used in calibration pipelines")
     (description "STScI tools and algorithms used in calibration pipelines.")
-- 
2.41.0

